client/elasticClientCommon.go

```diff
@@ -26,7 +27,7 @@ func exists(res *esapi.Response, err error) bool {
 	}()
 
 	if err != nil {
-		log.Warn("elasticClient.IndexExists", "could not check index on the elastic nodes:", err.Error())
+		log.Warn("elasticClient.IndexExists", "could not checkArgs index on the elastic nodes:", err.Error())
```

Comment:
what is checkArgs? This comment does not make much sense.

Reply:
changed.